packages/docusaurus-theme-common/src/components/Details/index.tsx

```diff
@@ -39,6 +39,23 @@ const Details = ({summary, children, ...props}: DetailsProps): JSX.Element => {
   // We use a separate prop because it must be set only after animation completes
   // Otherwise close anim won't work
   const [open, setOpen] = useState(props.open);
+  const toggle = (e: React.SyntheticEvent) => {
```

Comment:
nitpick: I'd rather not extract this method, it's only used in one place and it's not even meant to be reusable considering it's handling a click dom event. Extracting the code and naming it "toggle" makes this implicit. If yu really want to extract this (like for perf reasons using `useCallback` which in this case wouldn't improve anything), you'd rather name it `handleClick`.

Reply:
No problem, I wrote this a week ago and no idea why I extracted it back then 🤦‍♂️

Reply:
No problem, I wrote this a week ago and no idea why I extracted it back then 🤦‍♂️